src/app/shared/components/tab-card/style.scss

```diff
@@ -46,4 +46,8 @@
       margin: 0;
     }
   }
+
+  mat-ink-bar {
+    transition: 200ms cubic-bezier(0.35, 0, 0.25, 1);
```

Comment:
Can't this be merged with an already existing rule for the `.mat-ink-bar` a bit above?

Reply:
yes you are right. Haven't seen the `.mat-ink-bar`. Done